tls/s2n_security_policies.c

```diff
@@ -783,19 +783,18 @@ int s2n_security_policies_init()
             /* TLS1.3 does not include key exchange algorithms in its cipher suites,
```

Comment:
Hmm you may want to update/delete this comment if you're removing that ecc_extension_required variable.